packages/service/ni_measurement_plugin_sdk_service/measurement/service.py

```diff
@@ -204,7 +204,7 @@ def __init__(
         if not path.exists(service_config_path):
             raise RuntimeError(f"File does not exist. {service_config_path}")
 
-        with open(service_config_path) as service_config_file:
+        with open(service_config_path, "r", encoding="utf-8") as service_config_file:
```

Comment:
Does our C# implementation accept a BOM? You can do this with `encoding="utf-8-sig"`.

Reply:
The discovery service does accept the BOM, but the measurement service fails to load if it's there. I put in the `-sig`.